hessian-lite/src/test/java/com/alibaba/com/caucho/hessian/io/LocaleSerializerTest.java

```diff
@@ -1,3 +1,19 @@
+/*
```

Comment:
should hessian_lite be excluded from asf header check?

Reply:
>should hessian_lite be excluded from asf header check?

Yes, this is discussed in the dev@a